components/ItemSummary.tsx

```diff
@@ -29,7 +29,9 @@ export default function ItemSummary(props: Item) {
         {pluralize(props.score, "point")} by {props.userId}{" "}
         {timeAgo(new Date(props.createdAt))} ago •{" "}
         <a href={`/item/${props.id}`}>
-          Comments
+          {props.commentsCount
+            ? pluralize(props.commentsCount, "Comment")
+            : "Comments"}
```

Comment:
```suggestion
          {pluralize(props.commentsCount, "comment")}
```
`pluralize()` takes care of `commentsCount = 0`, and best to use lowercase "c".

Reply:
From what I saw from this code it would be resulting '0 comments' 

```
export function pluralize(unit: number, label: string) {
  return unit === 1 ? `${unit} ${label}` : `${unit} ${label}s`;
}
```
Could I modify the function with 
```
unit > 1  ? `${unit} ${label}s` : `${unit} ${label}`
```

Reply:
What would `pluralize(0, "comment")` return?

Reply:
> From what I saw from `pluralize()` code it would be resulting '0 comments'

Yes, that's correct and expected.

Reply:
For a further context, see https://www.quora.com/Why-do-we-say-use-a-plural-noun-when-we-specify-quantity-as-zero

Reply:
Wow I just know that 😂. 

Good to know though.